pantos/client/library/business/transfers.py

```diff
@@ -34,28 +40,33 @@ class TransferInteractorError(InteractorError):
     pass
 
 
-@dataclasses.dataclass
-class TransferTokensResponse:
-    """Response data for a new token transfer.
-
-    Attributes
-    ----------
-    task_id : uuid.UUID
-        The task ID of the chosen service node for the token
-        transfer.
-    service_node_address : BlockchainAddress
-        The address of the chosen service node for the token
-        transfer.
+class TransferInteractor(Interactor):
+    """Interactor for handling Pantos token transfers.
 
     """
-    task_id: uuid.UUID
-    service_node_address: BlockchainAddress
+    @dataclasses.dataclass
+    class TokenTransferStatusRequest:
+        """Request data for the status of a token transfer.
 
+        Attributes
+        ----------
+        source_blockchain : Blockchain
+            The token transfer's source blockchain.
+        service_node_address : BlockchainAddress
+            The address of the service node that is processing the
+            token transfer.
+        task_id : uuid.UUID
+            The task ID of the token transfer.
```

Comment:
We should make clear here that it's the service node's task ID (in contrast to any kind of global Pantos ID).